rollup.config.js

```diff
@@ -1,23 +1,71 @@
 import { version } from "./package.json";
 import git from "git-rev-sync";
+import { terser } from "rollup-plugin-terser";
+import typescript from "rollup-plugin-typescript2";
 
 let commitHash = "";
 
 try {
   commitHash = git.short();
+} catch (_) {}
+
+const OUTRO = `
+__info__.version = '${version}';
+__info__.date = '${new Date().toISOString()}';
+__info__.hash = '${commitHash}';
+`;
+
+/**
+ * Get the rollup config based on the arguments
+ * @param {string} format format of the bundle
+ * @param {string} generatedFileName generated file name
+ * @param {boolean} minified should it be minified
+ */
+function getConfigForFormat(format, minified = false) {
+  return {
+    file: minified ? `dist/o-spreadsheet.${format}.min.js` : `dist/o-spreadsheet.${format}.js`,
+    format,
+    name: "o_spreadsheet",
+    extend: true,
+    globals: { "@odoo/owl": "owl" },
+    outro: OUTRO,
+    plugins: minified ? [terser()] : [],
+  };
 }
-catch(_) {
+
+let output = [];
+let input = "";
+let plugins = [];
+console.log(process.argv);
+switch (process.argv[4]) {
+  // Only build iife version to improve speed
+  case "dev":
+    input = "build/js/index.js";
+    output = [
+      {
+        file: `build/o-spreadsheet-iife.js`,
+        format: "iife",
+        name: "o_spreadsheet",
+        extend: true,
+        globals: { "@odoo/owl": "owl" },
```

Comment:
does OUTRO take too muc time to add ? because I think we'll still use taht version in the deployment script. No need to generate all versions when we want to update the lib in Odoo

Reply:
No you can re-add it without any issue